miio/fan_miot.py

```diff
@@ -680,8 +680,7 @@ def __init__(
         lazy_discover: bool = True,
         model: str = MODEL_FAN_ZA5,
     ) -> None:
-        super().__init__(ip, token, start_id, debug, lazy_discover)
```

Comment:
Hmm, why not simply remove the whole constructor as it's not doing anything besides calling the base class one?